README.md

```diff
@@ -60,7 +60,7 @@ In order for the app to function correctly, the user must set up their own envir
 		c. ACCESS_KEY
 		d. ACCESS_SECRET
```

Comment:
Should we add in the consumer_key and consumer_secret? (not sure if I don't understand the technicality)
I think the bullet points arent alphabetical